src/modules/account/components/account-deposit/account-deposit.jsx

```diff
@@ -13,6 +13,13 @@ export default class AccountDeposit extends Component {
     address: PropTypes.string.isRequired
   }
 
+  static shapeShiftOnClick(e) {
```

Comment:
At least in my local testing, this guy does not need to be static. Try:

```<button onClick={e => this.shapeShiftOnClick(e)}  ...></button>```

Reply:
Or if you want to get really hardcore:

```
  shapeShiftOnClick(address, output) {
    return (e) => {
      e.preventDefault()
      const link = `https://shapeshift.io/shifty.html?destination=${address}&output=${output}`
      window.open(link, '1418115287605', 'width=700,height=500,toolbar=0,menubar=0,location=0,status=1,scrollbars=1,resizable=0,left=0,top=0')
    }
  }
```

And the onClick handlers would look like this:

```
onClick={this.shapeShiftOnClick(p.address, 'ETH')}
```

Reply:
I tried both of these approaches, but the linter still gives me the following error when I try to push my changes:
```
/Users/aaron/Documents/Augur/augur/src/modules/account/components/account-deposit/account-deposit.jsx
  21:20  error  Expected 'this' to be used by class method 'shapeShiftOnClick'  class-methods-use-this

✖ 1 problem (1 error, 0 warnings)
```

Reply:
We can screenshare if that would be helpful.

Reply:
Sure, let's look into this more after today's stand-up meeting.